moveos/moveos-stdlib/rooch-framework/sources/authenticator.move

```diff
@@ -0,0 +1,67 @@
+module rooch_framework::authenticator{
+
+    const SCHEME_ED25519:u64 = 0;
+    const SCHEME_MULTIED25519:u64 = 1;
+    const SCHEME_SECP256K1:u64 = 2;
+
+    const EUnsupportedScheme:u64 = 1000;
+
+    //TODO Migrate to other module, eg. rooch_framework::address
+    struct MultiChainAddress has copy, store, drop {
+        coin_type: u32,
```

Comment:
How does coin type + address_bytes express multi-chain addresses?

Reply:
coin type is slip standard https://github.com/satoshilabs/slips/blob/master/slip-0044.md